api/analyzers/csharp/analyzer.py

```
@@ -0,0 +1,137 @@
+import os
+import subprocess
+from pathlib import Path
+
+from multilspy import SyncLanguageServer
+from ...entities import *
```

Ruff (0.15.1)

[error] `from ...entities import *` used; unable to detect undefined names

(F403)